### DISCUSSION TITLE
Change rxSolve() options when using nlmixr2

### DISCUSSION THREAD

**Discussion by Percival24** [nlmixr2/nlmixr2#104] [Q&A] [answered]
Hi, 
I am using nlmixr2 (with saem) to estimate some parameters of a Pk model. 
Is there a way (throught nlmixr2) to access to the rxSolve option ? 
In particular, I would like to increase the maxsteps option. 
Thanks in advance, 
Percival

**Comment by mattfidler** [comment, marked as answer]
Hi @Percival24,

You can change the options in any of the control items.  

In general you can use `rxControl` inside the estimation method, or `rxControl` directly with the simulation method `"rxSolve"` as follows:

```r
fit <- nlmixr(model, data, "focei", foceiControl(rxControl=rxControl( maxsteps=100000L)))
fit <- nlmixr(model, data, "saem", saemControl(rxControl=rxControl( maxsteps=100000L)))
fit <- nlmixr(model, data, "nlme", nlmeControl(rxControl=rxControl( maxsteps=100000L)))
fit <- nlmixr(model, data, "rxSolve",rxControl( maxsteps=100000L))
```

This is now consistent between methods.

**Reply by Percival24** [reply]
Thanks a lot @mattfidler !!

**Comment by mattfidler** [comment]
I'm moving this to a discussion since it is more of a question than an issue.
